library/core/game_test/src/main/java/org/quiltmc/qsl/game_test/api/package-info.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * <h2>The Game Test API.</h2>
+ *
+ * <p>
+ * <h3>What is the Game Test API?</h3>
+ * The Game Test API is the test automation framework used by Minecraft as most test frameworks are not adapted to a game that's so dynamic.
+ * Unit testing presents very quickly its issues: you will quickly find out that testing while in a tick loop is very difficult.
+ * Thus Mojang made their own testing framework, watching the <a href="https://www.youtube.com/watch?v=TNkPE6NTNHQ">keynote by Henrik Kniberg</a>
+ * is highly recommended to grasp the why, and how it works.
+ * <p>
+ * The Game Test API exposed by this module is a bit different, the framework has been adapted to work better with mods, with some extensions.
```

Comment:
Different to what? Here's a revision based on what I think you meant:

```suggestion
 * The Game Test API exposed by this module is a bit different to what Vanilla provides. The framework has been adapted to work better with mods, with some extensions.
```